bin/deploy.sh

```diff
@@ -17,7 +17,7 @@ deploy_branch() {
                 --namespace=${K8S_NAMESPACE} \
                 --values ./helm_deploy/laa-fala/values/fala-uat.yaml \
                 --set image.repository="$ECR_ENDPOINT" \
-                --set image.tag="$IMAGE_TAG" \
+                --set image.tag="branch-$CIRCLE_SHA1" \
```

Comment:
should this be: "$BRANCH_RELEASE_NAME-$CIRCLE_SHA1"?

Otherwise it will be looking for something like 
"branch-k1j3hrjbf2krjg2k3j" 
instead of 
"EL-1234-thisthing-32r487yrtiu2wy"

Reply:
Clarified with Stephen that this is fine, we tag evrything branch or main and then use a CIRCLE_SHA1, it is the CIRCLE_SHA1 that is unique and so we can match an image to a deploy. resolving this